doc/reference/reference_lua/datetime.rst

```diff
@@ -118,7 +118,7 @@ Functions
                 -   0
 
             *   -   sec
-                -   Seconds. Value range: 0 - 60.
+                -   Seconds. Value range: 0 - 60. Leap second is supported, see a section :ref:`leap second <leap-second>`.
```

Comment:
Link to section `Leap second` doesn't work. Andrey, could you help please? @andreyaksenov

Reply:
Fixed, thanks!